src/tests/unit/tests/DetailsView/components/issues-table.test.tsx

```diff
@@ -122,6 +124,7 @@ class TestPropsBuilder {
             userConfigurationStoreData: {
                 bugService: 'gitHub',
             } as UserConfigurationStoreData,
+            instancesSection: NamedFC<InstancesSectionProps>('test', _ => null),
```

Comment:
Since we're rendering here, it might be a good idea to give a better name to this NamedFC. Like `SomeInstancesSection' or something ...

Reply:
Makes sense. Done